src/components/AlertDialog.tsx

```diff
@@ -25,32 +26,17 @@ export default function AlertDialog(props: AlertProps) {
 
   async function handleAgree() {
     try {
+      setOpen(false);
       await deleteById(props.id);
-      toast.success("Event Deleted Successfully! ", {
-        position: "top-right",
-        autoClose: 5000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: true,
-        draggable: true,
-        theme: "colored",
-      });
+      // force page refresh as there is no listener to change the events now, so refresh the page to show the event was successfully deleted
+      window.location.reload();
```

Comment:
It is better to change on the data state of the app which will cause a re-render, instead of reloading the entire page and requesting all of the files and APIs again.